page/static/css/theme-contao.css

```
@@ -521,6 +521,7 @@ option {
     padding: 0 25px 0 30px;
     margin: 0;
     font-weight: 300;
+    -webkit-appearance: none;
```

```suggestion
    appearance: none;
    -webkit-appearance: none;
```

Proper browsers do not need the prefix 😉 (Safari does).